### ISSUE TITLE
learning rate assignment needed to be correct

### ISSUE THREAD

**Issue by EdwardZH** [robosoup/Word2Vec#2]
Thank you for you providing this fast word2vec.
And I guess lr = (FLAGS.learning_rate * FLAGS.epochs_to_train - epoch) / FLAGS.epochs_to_train
need to be corrected to lr = FLAGS.learning_rate * (FLAGS.epochs_to_train - epoch) / FLAGS.epochs_to_train or you will have other suggestion.
